geode-web-api/src/main/java/org/apache/geode/rest/internal/web/security/RestSecurityService.java

```diff
@@ -46,6 +47,11 @@ public boolean authorize(String resource, String operation, String region) {
     return authorize(resource, operation, region, null);
   }
 
+  public void authorize(ResourcePermission permission) {
```

Comment:
Shouldn't this return a boolean?

Reply:
a boolean return value is required only if we are using the method in the PreAuthorize tag on the controller. e.g.
 @PreAuthorize("@securityService.authorize('DATA', 'READ')")

We are only using this method inside the controller body.